### PR TITLE
testing completed

### PR DESCRIPTION
**Pull request by MariaStavr**

### CODE REVIEW

File: life_expectancy/cleaning.py [hunk 1]
```diff
@@ -38,14 +38,14 @@ def clean_data(_df: pd.DataFrame, region_name: str) -> pd.DataFrame:
...
-def save_data(data) -> None:
+def save_data(data: pd.DataFrame) -> None:
     '''
     Saves the data to a new file.
     '''
     data.to_csv(DIR_PATH / "data" / SAVE_FILE_NAME, index=False)
 
 
-def parse_args():
+def parse_args_pt():
```
Comment (MrCordeiro) [inline life_expectancy/cleaning.py]: Question: why was this renamed "_pt"?

File: life_expectancy/tests/test_data_cleaning.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                    eu_life_expectancy_expected: pd.DataFrame) -> None:
+    """Test the `clean_data` function by comparing the output to the expected output"""
+    results = clean_data(eu_life_expectancy_raw, "PT")
+    pd.testing.assert_frame_equal(
+        results.reset_index(
+            drop=True), eu_life_expectancy_expected.reset_index(drop=True),
+        check_dtype=False)
+
+
+def test_save_data(monkeypatch: MonkeyPatch,
```
Comment (MrCordeiro) [inline life_expectancy/tests/test_data_cleaning.py]: pylint is giving you a warning here because you are not using monkeypatch. Should remove it.

File: life_expectancy/tests/test_data_cleaning.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    main("PT")
+    pt_life_expectancy_actual = pd.read_csv(
+        OUTPUT_DIR / "pt_life_expectancy.csv"
+    )
+    pd.testing.assert_frame_equal(
+        pt_life_expectancy_actual, pt_life_expectancy_expected
+    )
+
+
+@pytest.fixture(scope="session")
```
Comment (MrCordeiro) [inline life_expectancy/tests/test_data_cleaning.py]: Pylint is giving you a warning because of the way pytest fixtures work.
A good practice is to move the fixtures into conftest.py so they are not in the same module as the test code using the fixtures. Pytest load all fixtures on the `conftest.py` module by default, so this location is also useful for sharing fixtures among test modules.

**Review by MrCordeiro — CHANGES_REQUESTED** [review]
I just gave 2 little pointers so @GoncaloJardim also has something to do. But it's looking good overall! Isn't testing fun? 🤪

File: life_expectancy/cleaning.py (hunk 1 above)
Comment (MariaStavr) [reply life_expectancy/cleaning.py]: I thought it could cause a conflict when I patched the method 'parse_args()' since parse_args is also called inside the method. I reverted it though as it is unnecessary.

**Review by GoncaloJardim — APPROVED** [review]

**Review by MrCordeiro — APPROVED** [review]
